packages/grid/x-data-grid-premium/src/tests/rowGrouping.DataGridPremium.test.tsx

```diff
@@ -814,6 +815,32 @@ describe('<DataGridPremium /> - Row Grouping', () => {
           'Custom leaf',
         ]);
       });
+
+      it('should correctly pass `hasFocus` to `renderCell` defined on the leafColDef', () => {
```

Comment:
Link to the issue might be helpful in the future for more context on the problem :)

```suggestion
      // See https://github.com/mui/mui-x/issues/7949
      it('should correctly pass `hasFocus` to `renderCell` defined on the leafColDef', () => {
```